### ISSUE TITLE
 Low keypoint AP result on COCO ?

### ISSUE THREAD

**Issue by my-hello-world** [facebookresearch/maskrcnn-benchmark#752]
## ❓ Questions and Help

Hello, did anyone train or finetune mask-rcnn on COCO?
I just found the result can not match the one reported  with this implementation（64.3）.
For res-50-fpn, I trained from imagenet pretrained weight, and I got 58.8 keypoint AP on validation set,
For model finetuned from coco pretrained weight, I got 60.9 AP.

Up to now the difference I found is that I train the model with batchsize as 2 on 1 gpu while mask-rcnn paper uses batchsize as 16 on 8 gpu. 
and

> SOLVER:
>   BASE_LR: 0.0025
>   WEIGHT_DECAY: 0.0001
>   STEPS: (390000,400000)
>   MAX_ITER: 450000
The reason that MAX_ITERis not equal to 720k is because I found that since 400k, every 100k iterations have been tested,  ap stopped rising. 
any suggestions ?

BTW,What's the AP with "e2e_keypoint_rcnn_R_50_FPN_quick.yaml"? 

Thanks!
